README.md

```diff
@@ -38,29 +60,47 @@ In order for this to work, you'll need to add the following environment variable
 The script will bind the default arango port 8529 of the container to `$ARANGO_DB_PORT` on the host. To verify, you can open
 
 ```
-localhost:${ARANGO_DB_PORT}
+http://localhost:${ARANGO_DB_PORT}
 ```
 
 in your browser and access the dashboard using `${ARANGO_DB_ROOT_PASSWORD}`.
 
-### Lint
+## Workflow
+
+<!-- ### Lint
 
 In the project root, run
 
-> > npm run lint
+> > npm run lint -->
 
 ### Tests
 
-Note: Make sure nx cli is installed globally to run the following command.
+Note: Make sure nx cli is installed globally to run the following commands.
+
+To run the tests for a specific project, run
 
-> > nx test <project-name>
+> > nx test <project-name>.
```

Comment:
Should we just stick to `npm` run instead of `nx` for consistency?  Or does nx provide additional functionality here?